subprojects/docs/src/samples/userguide/dependencyManagement/troubleshooting/cache/changing/build.gradle

```diff
@@ -2,13 +2,13 @@ apply plugin: 'java-library'
 
 repositories {
     mavenCentral()
```

Comment:
❌ Are our samples tests already configured to use the repository mirrors? If not, we should definitely add it, as mavenCentral can have issues too.

Reply:
Will double check, I had assumed that was the case, but not 100% sure

Reply:
I'm pretty sure they are not, since there is a literal `mavenCentral` in there and our repo [mirroring init script](https://github.com/gradle/gradle/blob/b25f675dc0346b32633b404ba8b5bb5c4f0de9bf/subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/RepoScriptBlockUtil.groovy#L104) is not used in that test. I only introduced it recently in the performance tests.

Reply:
Be aware that the code snippets you are changing here are rendered one to one in the user guide. We cannot use the mirror as people will copy-paste the code. Alternatively, you will have to exclude the repository from being rendered in the user guide. However, I'd love to keep the samples "self-sufficient" meaning users can basically copy them and put them into their build scripts.